### PR TITLE
EPLT-926: white label sendr

### PR DESCRIPTION
**Pull request by gusk**
- Apply Checkr Brand guidelines.
- Leverage reuse of CSS components using Tailwind
- Copy defined within the translation files
- Minimize changes that might conflict with upstream

### CODE REVIEW

**Comment by renjiexu** [comment]
Is there a snapshot?

**Comment by gusk** [comment]
snapshots: https://drive.google.com/drive/folders/1WhP8_ksm62sh2l62gIYmxhCE6jxuGdeg

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@ ios/send-ios/assets/vendor.js
 ios/send-ios.xcodeproj/project.xcworkspace/xcuserdata/*
 ios/send-ios.xcodeproj/xcuserdata/*
 test/integration/downloads
+package-lock.json
```
Comment (charli-watson) [inline .gitignore]: `package-lock` should generally be committed

Comment (gusk) [reply .gitignore]: should't the deployment pipeline be responsible of installing & locking dependencies as opposed to devs?

Comment (charli-watson) [reply .gitignore]: committing it should ensure that what we deploy has the same dependencies that we tested with/developed on. Maybe a case for a larger discussion?

Comment (gusk) [reply .gitignore]: removing package-lock.json to allow npm ci to build from package-lock.json (as opposed to building from package.json)

File: package-lock.json [hunk 2]
```diff
@@ -7573,8 +7573,7 @@
           "version": "2.1.1",
           "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-2.1.1.tgz",
           "integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8=",
-          "dev": true,
-          "optional": true
```
Comment (cjford) [inline package-lock.json:7577]: Were these changes intentional or just happened when you re-generated the lock file? We might want to just leave them out if we're not consciously changing dependencies. Same thing happened to me due to running a different version of npm, fixed it by changing to v6.11.3 (node v10.17.0)

Comment (gusk) [reply package-lock.json:7577]: this is the lock after running `npm run build | Builds the production assets.` i'd assume it's ok have this committed

Comment (gusk) [reply package-lock.json:7577]: after looking into `dev true` & `optional true`:
```If true then this dependency is either a development dependency ONLY of the top level module or a transitive dependency of one. This is false for dependencies that are both a development dependency of the top level and a transitive dependency of a non-development dependency of the top level.```

Comment (cjford) [reply package-lock.json:7577]: Yep, just wanted to reduce upstream diff if possible but agreed I think it's fine.

**Review by cjford — APPROVED** [review]
